content/src/policies/__id/post.rego

```diff
@@ -1,19 +1,33 @@
 package peoplefinder.POST.api.users.__id
 
+# Policy for updating an employee's deparment and title in PeopleFinder
+#
+# The user is Allowed if:
+# - User is an admin
```

Comment:
maybe you could insert an "or" here, to make it clear that either or?

or, `# The user is Allowed if either is true:`

Reply:
@glennblock

Reply:
Done in both templates. Example of the format below.

```
# The user is Allowed if:
# - User is an admin
# - OR User is the manager or in the management chain of the employee```